src/pyosmeta/parse_issues.py

```diff
@@ -161,7 +161,11 @@ class ReviewModel(BaseModel):
     reviewer_2: dict[str, str | None] = {}
     archive: Optional[str] = None
     version_accepted: Optional[str] = None
-    date_accepted: Optional[str] = None
+    date_accepted: Optional[str] = Field(
```

Comment:
here i added an alias to allow us to skip the step of cleaning up the field name